src/main/java/org/kitteh/irc/client/library/feature/twitch/messagetag/Login.java

```diff
@@ -0,0 +1,23 @@
+package org.kitteh.irc.client.library.feature.twitch.messagetag;
```

Comment:
Missing license header.

Reply:
This can be resolved by just running maven once, FYI :)

Reply:
I ran Maven install and it was not part of the build. Let me check for a license target.